### PR TITLE
Support PHP-Timer 4 and 5 for PHPUnit 9.2

### PR DESCRIPTION
**Pull request by Slamdunk**

### CODE REVIEW

File: src/Runners/PHPUnit/ResultPrinter.php [hunk 1]
```diff
@@ -107,7 +103,6 @@ class ResultPrinter
     public function __construct(LogInterpreter $results)
     {
         $this->results = $results;
-        $this->timer = new Timer();
```
Comment (Slamdunk) [inline src/Runners/PHPUnit/ResultPrinter.php:110]: There is no need to instantiate Timer for our usage, see

1. PHP-Timer 2: https://github.com/sebastianbergmann/php-timer/blob/2.1.2/src/Timer.php#L92
1. PHP-Timer 3: https://github.com/sebastianbergmann/php-timer/blob/3.1.4/src/Timer.php#L136
1. PHP-Timer 4: https://github.com/sebastianbergmann/php-timer/blob/4.0.0/src/ResourceUsageFormatter.php#L35
1. PHP-Timer 5: https://github.com/sebastianbergmann/php-timer/blob/5.0.0/src/ResourceUsageFormatter.php#L35

**Comment by Slamdunk** [comment]
The failures on the build appear the same as the ones in master branch, unrelated to this very PR.

**Comment by bcremer** [comment]
Thanks for opening the PR.
Stumbled about this incompatibility just today. 

```
  Problem 1
    - Conclusion: remove brianium/paratest 4.0.0
    - Conclusion: remove phpunit/php-timer 3.1.4
    - Conclusion: don't install phpunit/php-timer 3.1.4
    - Installation request for brianium/paratest ^4.0 -> satisfiable by brianium/paratest[4.0.0].
    - Installation request for phpunit/phpunit ^9.2 -> satisfiable by phpunit/phpunit[9.2.0, 9.2.1, 9.2.2].
    - brianium/paratest 4.0.0 requires phpunit/php-timer ^2.0|^3.0 -> satisfiable by phpunit/php-timer[3.1.4, 2.0.0, 2.1.0, 2.1.1, 2.1.2, 3.0.0, 3.1.0, 3.1.1, 3.1.2, 3.1.3].
    - Can only install one of: phpunit/php-timer[4.0.0, 2.0.0].
```

This PR looks good for me.

**Comment by Slamdunk** [comment]
Ping @andreasschroth

**Comment by andreasschroth** [comment]
Apologies for the late reply and thank you for the PR.

**Comment by Slamdunk** [comment]
No need to apologize, you don't owe anything to anyone, FOSS maintenance is expensive and without profit.

But maybe you can look for collaborators to share the burden ;)

**Pull request merged**